source/components/DatasetSelector.js

```diff
@@ -81,6 +86,7 @@ export default class DatasetSelector extends React.Component {
 DatasetSelector.propTypes = {
   labels: React.PropTypes.array,
   onDatasetSelected: React.PropTypes.func,
+  onCustomDataset: React.PropTypes.func,
```

Comment:
https://github.com/yannickcr/eslint-plugin-react/issues/528 doesn't seem like it yet :(